src/platform/audio/milesaudiomanager.cpp

```diff
@@ -60,6 +60,7 @@ MilesAudioManager::~MilesAudioManager()
     Unselect_Provider();
```

Comment:
indeed this is correct to pseudo, missed this when reviewing miles :X

can you also replace first if with Release_Bink_Handle and these 3 calls with Close_Device

Reply:
Done